### PR TITLE
[Multi] Fixes some CSS lint issues

### PR DESCRIPTION
**Pull request by prudho**
## Description
This PR fixes several small CSS issues raised by my local linter ([stylelint](https://stylelint.io)). Most of them are duplicate properties or CSS selectors, and I just fixed the most obvious.

### CODE REVIEW

**Comment by lubber-de** [comment]
You may aswell consider https://github.com/Semantic-Org/Semantic-UI/pull/6401 to get rid of all 0em, 0px when refactoring the less files. I dont know, if you can configure this in styleint or if we should create an additional issue for this :thinking:

**Comment by prudho** [comment]
For now, this is just a first shot, fired while I was making some linting implementation for #29. This is just the top of the iceberg on lint issues, and there's many fixes and tweaks to provide yet, before passing the stylelint check. We should not create lining issues while linting is not set up and running.

File: src/definitions/modules/modal.less [hunk 1]
```diff
@@ -479,9 +477,9 @@
   }
 }
 
-/* mini */
-.ui.small.modal > .header:not(.ui) {
-  font-size: @miniHeaderSize;
+/* Tiny */
+.ui.tiny.modal > .header:not(.ui) {
+  font-size: @tinyHeaderSize;
```
Comment (hammy2899) [inline src/definitions/modules/modal.less]: What does changing this to `tiny` fix?

Comment (prudho) [reply src/definitions/modules/modal.less]: `mini` was defined two times, and `tiny` was missing from the file. This was a typo.

Comment (hammy2899) [reply src/definitions/modules/modal.less]: Ok cool.

**Pull request merged**